Lombiq.Tests.UI.Shortcuts/Controllers/InteractiveModeController.cs

```diff
@@ -2,11 +2,18 @@
 using Lombiq.Tests.UI.Shortcuts.Services;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Lombiq.Tests.UI.Shortcuts.Controllers;
 
 [AllowAnonymous]
 [DevelopmentAndLocalhostOnly]
+[SuppressMessage(
+    "Major Code Smell",
+    "S6934:A Route attribute should be added to the controller when a route template is specified at the action level",
+    Justification = "Using attribute routing on the Controller breaks the behavior and will result in a 404." +
+    "[controller]/{action=Index} on the controller will not hit the Index action method.")]
```

Comment:
This is most possibly because the pattern is wrong. See my comments on the Training Demo PR.